include/settings/types/LayerIndex.h

```diff
@@ -8,6 +8,7 @@
 
 #include <fmt/format.h>
 
+#include "geometry/Polygon.h"
```

Comment:
Very same remark here, not used in this header so should not be required.

Reply:
https://github.com/Ultimaker/CuraEngine/pull/2082/commits/57d357d497f7e824cb0c8526799b5ed1978fb1d5